recipes/islandpath/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+mkdir -p $PREFIX/islandpath
+cp -r . $PREFIX/islandpath
+mkdir -p $PREFIX/bin
+ln -s $PREFIX/islandpath/Dimob.pl $PREFIX/bin/islandpath
+chmod +x $PREFIX/bin/*
+export HOME=/tmp 
+cpanm --force TAP::Harness::Env ExtUtils::Helpers
+cpanm MooseX::Singleton
```

Comment:
ideally this should be a separate perl package which you depend on

Reply:
perl-moosex-singleton  is now added as a bioconda package